Source/WebKit/UIProcess/API/APIPageConfiguration.h

```diff
@@ -507,6 +509,7 @@ class PageConfiguration : public ObjectImpl<Object::Type::PageConfiguration> {
         RefPtr<WebKit::WebPageGroup> pageGroup;
         WeakPtr<WebKit::WebPageProxy> relatedPage;
         std::optional<OpenerInfo> openerInfo;
+        std::optional<WebCore::Site> openedSite;
```

Comment:
If this cannot be null, should we just make this non-optional and default to `WebCore::Site(aboutBlankURL())` then?